docs/README.md

```diff
@@ -12,11 +14,11 @@ When running a EDC connector from the Tractus-X EDC repository there are three s
 
 The three supported setups are.
 
-- Setup 1: In Memory & Azure Vault
-  - [Control Plane](../edc-controlplane/edc-controlplane-memory/README.md)
+- Setup 1: Pure in Memory
+  - [Control Plane](../edc-controlplane/edc-runtime-memory/README.md)
     - [IDS DAPS Extensions](https://github.com/eclipse-edc/Connector/tree/main/extensions/common/iam/oauth2/daps)
     - In Memory Persistence done by using no extension
-    - [Azure Key Vault Extension](https://github.com/eclipse-edc/Connector/tree/main/extensions/common/vault/azure-vault)
+    - In Memory Keyvault with seedable secrets. **Not intended for production use!**
```

Comment:
I would put `**Not intended for production use!**` directly at the end of the first pulled point: `Setup 1: Pure in Memory`